config/environment.js

```diff
@@ -29,30 +29,31 @@ module.exports = function(environment) {
         }]
       ]
     },
-
-    web3ProviderUrl: "https://parity.kosmos.org:8545",
-    ethereumChain: "testnet",
+    contractMetadata: {},
     ipfs: {
       host: 'localhost',
       port: '5001',
       protocol: 'http'
     }
   };
+  if (process.env.KREDITS_CONTRACT_ADDR) {
```

Comment:
We use config files in Ember, so it's clear what is being built. Otherwise you end up with a build that completely hides where its config comes from.